Trio/Sources/APS/DeviceDataManager.swift

```diff
@@ -411,6 +415,25 @@ extension BaseDeviceDataManager: PumpManagerDelegate {
             bolusTrigger.send(false)
         }
 
+        /// The new suspended and scheduledBasal PassthroughSubject var's are set here based on the PM's basalDeliveryState.
```

Comment:
This is a comment that explains a code change, but does not need to be here going forward.
Please remove.